vtex-apps/checkout-io-tests
# CHK-113: Display tests results on Healthcheck

#### What does this PR do?

Now, running tests with `vtex test e2e --token --workspace` will `POST` the result of each spec (i.e.: each test file) to the Checkout IO Tests service, which will send the results to a Checkout IO module on Healthcheck, under the beta environment.

#### How to test it?

1. Login on `checkoutio` account and use `w0e2e` workspace.
1. Run `vtex test e2e --token --workspace`
1. Check if the four specs were displayed on the [Checkout IO Module on Healthcheck](http://monitoring.vtex.com/healthcheck/results?module=Checkout%20IO&environment=beta) when the tests are complete (it doesn't matter if they fail)
1. Click in any spec in order to go to the evidence page and assert that the "details" link, under the "start date", redirects to a Mochawesome report

#### Screenshots

<img width="600" alt="Screen Shot 2020-09-30 at 16 21 09" src="https://user-images.githubusercontent.com/26108090/94735574-57756100-0341-11eb-9ba8-f42ef48b5d6f.png">

<img width="600" alt="Screen Shot 2020-09-30 at 16 19 14" src="https://user-images.githubusercontent.com/26108090/94735513-4298cd80-0341-11eb-89fa-c6435bf5a54d.png">

<img width="600" alt="Screen Shot 2020-09-30 at 16 23 00" src="https://user-images.githubusercontent.com/26108090/94735774-9f948380-0341-11eb-8419-1e1757496c2c.png">

#### Notes

I believe it's better to keep the module on the beta environment for a while, until the tests are fixed.
